composer/core/engine.py

```diff
@@ -182,6 +182,11 @@ def __init__(
 
         self.algorithm_passes: List[passes.AlgorithmPass] = _get_default_passes()
         if algorithm_passes is not None:
+            # Wrap in list if not already a list or if it's a length 2 list specifying a single
```

Comment:
Is there a particular reason you're changing it to allow a two element list?

Reply:
See extra test I added. If you pass args for a single call to register pass, it should wrap in a list. However, if you pass [algopass, idx]  instead of (algopass, idx), it will fail wrapping and error out. This just makes it slightly more robust